ownership-chain/pallets/asset-metadata-extender/src/benchmarking.rs

```diff
@@ -43,4 +43,34 @@ mod benchmarks {
 			Some(token_uri)
 		);
 	}
+
+	#[benchmark]
+	fn update_token_uri_extension(
+		t: Linear<0, { <T as Config>::MaxTokenUriLength::get() }>,
+		u: Linear<0, { <T as Config>::MaxUniversalLocationLength::get() }>,
+	) {
+		let claimer: T::AccountId = whitelisted_caller();
+		let universal_location: UniversalLocationOf<T> = vec![1u8; u as usize].try_into().unwrap();
+		let token_uri: TokenUriOf<T> = vec![1u8; t as usize].try_into().unwrap();
+
+		{
+			AssetMetadataExtender::<T>::create_token_uri_extension(
+				claimer.clone(),
+				universal_location.clone(),
+				token_uri,
+			)
+			.unwrap();
+		};
+
+		let new_token_uri: TokenUriOf<T> = vec![2u8; t as usize].try_into().unwrap();
+
+		#[block]
+		{
+			let _ = AssetMetadataExtender::<T>::update_token_uri_extension(
```

Comment:
this doesn't handle the error case properly. if this call results in `Error` some storage entries won't be altered which will lead to this block recording invalid number of writes/reads. 

either handle the error with `unwrap()` or add `assertions` outside of the block that the storage is altered. If you add `unwrap` it will panic on `Error` letting us know it failed, but benchmark won't be calculated. So probably adding `asserts` is a better choice

Reply:
it's ok to panic ... if it errors there's a critic problem and we don't want to calculate a benchmark